### PR TITLE
Add new required filter.type

### PR DESCRIPTION
**Pull request by jwildfire**

### CODE REVIEW

**Comment by jwildfire** [comment]
@samussiah h can you QC/Code review this little bit of the refactor please? See #4 for details. Example [here](http://bl.ocks.org/jwildfire/c5355b066da3c81de480f73876860eb9)

File: src/chart/controls/filters/custom/init.js [hunk 1]
```diff
@@ -62,11 +63,8 @@ export function init(chart) {
                     return ['NA', '', ' '].indexOf(di) === -1; }); })
         .enter()
         .append('option')
-        .html(function(d) {
-                return  '<span><i class = "icon-remove icon-white fa fa-times"></i></span>'
-                    +   (['NA', '', ' '].indexOf(d) > -1 ? '[None]' : d); })
-        .attr('value', function(d) {
-            return d; })
+        .html(function(d) {return ['NA', '', ' '].indexOf(d) > -1 ? '[None]' : d})
```
Comment (samussiah) [inline src/chart/controls/filters/custom/init.js]: Those values are filtered out on line 63, no?

**Comment by jwildfire** [comment]
@samussiah 2 new commits in. Look good?

**Comment by jwildfire** [comment]
Note to self: still need to update default filters with filterType

**Pull request merged**